include/task.h

```diff
@@ -1683,6 +1683,16 @@ configSTACK_DEPTH_TYPE uxTaskGetStackHighWaterMark2( TaskHandle_t xTask ) PRIVIL
                                         uint32_t * pulIdleTaskStackSize ); /*lint !e526 Symbol not defined as it is an application callback. */
 #endif
 
+#if ( configUSE_MALLOC_FAILED_HOOK == 1 )
```

Comment:
This change seems unrelated?

Reply:
Sorry both pull request was done, last commit should be in other pull request.

Reply:
Can you remove these changes from this PR?

Reply:
How to do that ?

Reply:
In general, you can put those changes in 2 separate branches and raise 2 separate PRs. However, malloc failed hook is not invoked from tasks.c and therefore, task.h does not seem the right place to put it. Can you just remove that change and keep only the  identical branches change?

Reply:
Thank you. Would you please update the comment also as I suggested below?